examples/helpers/mail/Example.java

```diff
@@ -149,6 +149,15 @@ public static Mail buildKitchenSink() {
     Setting bypassListManagement = new Setting();
     bypassListManagement.setEnable(true);
     mailSettings.setBypassListManagement(bypassListManagement);
+    Setting bypassSpamManagement = new Setting();
```

Comment:
If the spam and bounce management filters cannot be used with the list management filter, can you call that out here in the example?

Reply:
yep, added.

Reply:
Thank you!